lusid-jam/refreshing_token.py

```diff
@@ -0,0 +1,31 @@
+from collections import UserString
+from datetime import datetime, timedelta, time
+import os
+
+class RefreshingToken(UserString):
+
+    def __init__(self, access_token=os.path.join(os.getenv("FBN_ACCESS_TOKEN_FILE"))):
```

Comment:
Can we rename this variable to access_token_location please? This makes me think that I am providing the actual token.

Reply:
Renamed to access_token_location